api/sales_dashboard/forms.py

```diff
@@ -28,6 +33,29 @@ def save(self, organisation, commit=True):
         organisation.subscription.save()
 
 
+class StartTrialForm(forms.Form):
+    max_seats = forms.IntegerField()
+    max_api_calls = forms.IntegerField()
+
+    def save(self, organisation, commit=True):
```

Comment:
Typehints please!

```suggestion
    def save(self, organisation: Organisation, commit: bool = True):
```

Reply:
Also, this (and the similar save method on the EndTrialForm) should probably just call a new method on the `Subscription` class like `start|end_trial(max_seats: int, max_api_calls: int)`. That way, we can easily unit test the method on the subscription class.